app/ide-desktop/lib/dashboard/src/authentication/src/index.tsx

```diff
@@ -47,6 +49,12 @@ export function run(
   if (root == null) {
     logger.error(`Could not find root element with ID '${ROOT_ELEMENT_ID}'.`);
   } else {
+    // FIXME[sb]: This is a temporary workaround and will be fixed
+    // when IDE support is properly integrated into the dashboard.
+    const ide = document.getElementById(IDE_ELEMENT_ID);
+    if (ide != null) {
+      ide.style.display = "none";
+    }
```

Comment:
@somebody1234 Commenting out these lines seems to make the error non-reproducible. I think they may be responsible.

At a glance, it looks like we never reset the `ide.style.display` back to `""`. So it remains `"none"` after authentication is complete and we do `appInstance.run()`. So EnsoGL tries to render to a non-visible element, which I'm assuming is not possible.

Making the IDE visible prior to calling `appInstance.run()` seems to fix it:
```diff
diff --git a/app/ide-desktop/lib/content/src/index.ts b/app/ide-desktop/lib/content/src/index.ts
index 3fc612a43..34d3db0e7 100644
--- a/app/ide-desktop/lib/content/src/index.ts
+++ b/app/ide-desktop/lib/content/src/index.ts
@@ -143,7 +143,9 @@ class Main {
                 ) {
                     const hideAuth = () => {
                         const auth = document.getElementById('dashboard')
+                        const ide = document.getElementById('root')
                         if (auth) auth.style.display = 'none'
+                        if (ide) ide.style.display = ''
                     }
                     /** This package is an Electron desktop app (i.e., not in the Cloud), so
                      * we're running on the desktop. */
```